### ISSUE TITLE
Installation Errors v2.3.1

### ISSUE THREAD

**Issue by alvarnell** [JohnCoates/Aerial#1173]
Reporting as requested:
<img width="532" alt="Screen Shot 2021-04-27 at 13 33 27" src="https://user-images.githubusercontent.com/4943704/116309492-acede100-a75d-11eb-9621-5f925c47082c.png">
<img width="532" alt="Screen Shot 2021-04-27 at 13 33 39" src="https://user-images.githubusercontent.com/4943704/116309486-ab241d80-a75d-11eb-8faa-4abbca7fa1dd.png">

**Comment by glouel** [comment]
Hey there

Can you give me a bit more context ? Is this a fresh install or when trying to upgrade ?

Is there anything particular about your mac or your user account ? (some form of MDM, unpriviledged account or some specific software on it etc)

That may help me narrow it down, as I don't think I've had such a report yet.

**Comment by alvarnell** [comment]
On Apr 27, 2021, at 14:13, Guillaume Louel ***@***.***> wrote:

         
Powered by Mailbutler <https://www.mailbutler.io/?utm_source=watermark&utm_medium=email&utm_campaign=watermark-essential-email>, the email extension that does it all
> Hey there
> 
> Can you give me a bit more context ? Is this a fresh install or when trying to upgrade ?
> 
Upgrade from the menubar icon.
> Is there anything particular about your mac or your user account ? (some form of MDM, unpriviledged account or some specific software on it etc)
> 
No. Admin account. No MDM. macOS Mojave 10.14.6 (18G9208)
> That may help me narrow it down, as I don't think I've had such a report yet.
> 
Where is the download directory that it uses?
Would you like me to try a fresh install?

-Al-

**Comment by glouel** [comment]
That's so weird... I think the issue is the unzip failed or something, or maybe the download failed for some reason. 

To answer your question, the folder where it downloads is : ~/Library/Application Support/AerialUpdater/

Can you check if there's a Aerial.saver.zip in there ? 

You could delete the content of that directory too and try again.

Sorry about the issue, definitely never seen that one.

**Comment by alvarnell** [comment]
On Apr 27, 2021, at 14:22, Guillaume Louel ***@***.***> wrote:

         
Powered by Mailbutler <https://www.mailbutler.io/?utm_source=watermark&utm_medium=email&utm_campaign=watermark-essential-email>, the email extension that does it all
> That's so weird... I think the issue is the unzip failed or something, or maybe the download failed for some reason.
> 
> To answer your question, the folder where it downloads is : ~/Library/Application Support/AerialUpdater/
> 
> Can you check if there's a Aerial.saver.zip in there ?
> 
> You could delete the content of that directory too and try again.
> 
Same result.

**Comment by glouel** [comment]
Really puzzling, well, as it's getting late here, you can try upgrading manually, download this file : 

https://github.com/JohnCoates/Aerial/releases/download/v2.3.1/Aerial.saver.zip

Unzip and double click .saver, when prompted to install, install for your user only (not all users) and you should be set. I'll keep an eye on the issue and try to see what could have happened. Thanks for the report !

**Comment by alvarnell** [comment]
Problem solved.

Manual update didn't work either and in checking around I found a few other issues with saving files. Get info on various directories and files revealed nothing unusual, but clearly something was wrong with permissions. Last time that happened I had to disable SIP and then Apply home directory permissions to all enclosed items.... That solved the problem. Doing same without disabling SIP has not worked.

Sorry to have bothered.

-Al-

         
Powered by Mailbutler <https://www.mailbutler.io/?utm_source=watermark&utm_medium=email&utm_campaign=watermark-essential-email>, the email extension that does it all
